TLM/TLM/TLM.csproj

```diff
@@ -557,11 +557,8 @@
       $(PostBuildEventDependsOn);
       PostBuildMacros;
     </PostBuildEventDependsOn>
-    <PostBuildEvent>
-rem set "DEPLOYDIR=D:\Games\Steam\steamapps\workshop\content\255710\1637663252\"
+    <PostBuildEvent>rem set "DEPLOYDIR=D:\Games\Steam\steamapps\workshop\content\255710\1637663252\"
```

Comment:
Hard-coded path

Reply:
It was there all along. I assumed someone wants to see how TMPE works when it is in the steamapps directory as opposed to the local mod directory.

I can get rid of it if you want